.cargo/mutants.toml

```diff
@@ -1,15 +1,15 @@
 additional_cargo_args = ["--all-features"]
 gitignore = true
 examine_globs = [
-	"payjoin/src/core/ohttp.rs", 
-	"payjoin/src/core/uri/*.rs", 
-	"payjoin/src/core/receive/**/*.rs", 
-	"payjoin/src/core/send/**/*.rs"
+	"payjoin/src/**/*.rs"
+]
+exclude_globs = [
 ]
-exclude_globs = []
 exclude_re = [
 	"impl Debug",
+	"impl fmt::Debug",
 	"impl Display",
+	"impl fmt::Display",
```

Comment:
Can regex be used here? something like "impl (fmt::)?(Debug|Display)" ?

Reply:
It didn't work initially but I haven't given the cargo mutants a thorough read in this part of the docs

Reply:
Ok, nvm that worked, I must have simply formatted my regex wrong when I tried